### PR TITLE
Hotfix for the issue in waking the app.

### PR DESCRIPTION
**Pull request by ricdezen**
There was a big issue. As always it probably passed because of my complete inability to spot errors before actually testing stuff myself. The library has not been using a broadcast for urgent messages for a fairly long time, I suppose this was never fixed here. There are two TODOs, because this is just a provvisory hotfix for the problem, the structure will have to change once we sync with the shared library.

### CODE REVIEW

File: app/src/main/java/ingsw/group1/findmyphone/activity/AlarmAndLocateResponseActivity.java [hunk 1]
```diff
@@ -19,7 +21,8 @@
     public MediaPlayer mediaPlayer;
 
     /**
-     * This activity is created in all situations, for each request, so it needs to be executed also when screen is shut
+     * This activity is created in all situations, for each request, so it needs to be executed
```
Comment (ricdezen) [inline app/src/main/java/ingsw/group1/findmyphone/activity/AlarmAndLocateResponseActivity.java:24]: A regà CTRL+ALT+L ogni tanto premetelo figa

**Review by turcato-niccolo — APPROVED** [review]

File: app/src/main/java/ingsw/group1/findmyphone/activity/AlarmAndLocateResponseActivity.java (hunk 1 above)
Comment (turcato-niccolo) [reply app/src/main/java/ingsw/group1/findmyphone/activity/AlarmAndLocateResponseActivity.java:24]: Ti ga rason

Comment (ricdezen) [reply app/src/main/java/ingsw/group1/findmyphone/activity/AlarmAndLocateResponseActivity.java:24]: Altrimenti cercate di settarlo nella schermata di commit, è più comodo

Comment (turcato-niccolo) [reply app/src/main/java/ingsw/group1/findmyphone/activity/AlarmAndLocateResponseActivity.java:24]: Sì io ho iniziato a farlo quando me l'hai fatto notare tempo fa

**Pull request merged**